dev/Common/ReunionTelemetry.h

```diff
@@ -0,0 +1,55 @@
+﻿#pragma once
+
+#ifndef __REUNIONTELEMETRY_INCLUDED
+#define __REUNIONTELEMETRY_INCLUDED
+
+#ifdef __WIL_TRACELOGGING_H_INCLUDED
+#error "WIL Tracelogging.h must not be explicitly included when including this file"
+#endif
+
+#define _REUNION_PACKAGE_VER_RESOURCEID             10001
+#define _REUNION_EXPERIMENTATION_FLAG_RESOURCEID    10002
+
+static std::wstring s_ReunionPackageVer = L"";
+static std::wstring s_ExperimentationLevel = L"not_initialized";
+
+wchar_t* REUNION_PACKAGE_VER()
+{
+   if(s_ReunionPackageVer.empty())
+   {
+       HMODULE resourceDll = LoadLibraryW(L"ProjectReunionTelemetryResource.dll");
```

Comment:
Does this code need to handle errors? (Should it check if the return is `NULL`? )

Reply:
If the process is unable to load the resource dll, we must have a way of signaling the configuration error.

Reply:
Why? If it can't load the resource DLL it should act as if it wasn't there and the package version and configuration settings aren't present.  That is, "missing the DLL means not configured."